larq_zoo/utils.py

```diff
@@ -34,7 +42,7 @@ def get_current_epoch(output_dir):
     try:
         with open(os.path.join(output_dir, "stats.json"), "r") as f:
             return json.load(f)["epoch"]
-    except:
+    except Exception:
         return 0
```

Comment:
Should we not raise the original Exception here? We can just replace `return 0` by `raise`, right?

Reply:
We want to return zero, because in particular an exception would be raised if the file doesn't exist, as is the case at the start of training.

Reply:
Ah my bad, I didn't realise that it returns the latest epoch :+1: